proposals/exception-handling/Exceptions.md

```diff
@@ -164,10 +164,10 @@ after possible block parameters were popped.
 
 Then catch clauses are tried in the order they appear in the catching try block,
 until one matches. If a matching catch clause is found, control is transferred
-to the label of that catch clause. In case of `catch` or `catch_ref`, the
-arguments of the exception are pushed back onto the stack. For `catch_ref` and
-`catch_all_ref`, an exception reference is then pushed to the stack, which
-represents the caught exception.
+to the label of that catch clause. In case of `catch`, the arguments of the
```

Comment:
This change is not correct. Now the phrasing says that `catch_ref` pushes *only* the ref. But it pushes the arguments *and* the ref.

Reply:
You're right; I was confused and the original text was correct.. Will revert this part.